evergreen/staticfiles/css/styles.css

```diff
@@ -118,14 +112,49 @@ footer {
     margin-left: 5%;
 }
 
+.likeButton button {
+    padding: 8px 16px;
+    border: 1px solid #4caf50;
+    border-radius: 4px;
+    cursor: pointer;
+    background-color: white;
+    color: #4caf50;
+    transition: all 0.3s ease;
+}
+
+.likeButton button.liked {
+    background-color: #4caf50;
+    color: white;
+}
+
+.likers-display {
+    font-size: 0.9em;
+    color: #666;
+    margin: 5px 0;
+}
+
+.upload-status {
+    margin-top: 10px;
+    padding: 10px;
+    border-radius: 4px;
+}
+
+.upload-status.success {
+    background-color: #dff0d8;
+    color: #3c763d;
+}
+
+.upload-status.error {
+    background-color: #f2dede;
+    color: #a94442;
+}
```

Comment:
I don't see this come up anywhere when uploading

Reply:
got rid of them, did not like them.